src/pke/lib/schemerns/rns-cryptoparameters.cpp

```diff
@@ -388,4 +388,51 @@ uint64_t CryptoParametersRNS::FindAuxPrimeStep() const {
     return GetElementParams()->GetRingDimension();
 }
 
+std::pair<double, uint32_t> CryptoParametersRNS::EstimateLogP(uint32_t numPartQ, double firstModulusSize,
+                                                              double dcrtBits, double extraModulusSize,
+                                                              uint32_t numPrimes, uint32_t auxBits) {
+    uint32_t sizeQ = numPrimes;
+    if (extraModulusSize > 0)
+        sizeQ++;
+
+    // Compute ceil(sizeQ/m_numPartQ), the # of towers per digit
```

Comment:
Remove m_ from the comment. Is there a reason why different types of casting are used on lines 399 and 400?

Reply:
Agreed. Will fix it in the next commit.